test/compile/mark-tick.test.ts

```diff
@@ -68,4 +68,22 @@ describe('Mark: Tick', function() {
       assert.deepEqual(props.yc, {scale: Y, field: 'Cylinders'});
     });
   });
+
+  describe('with size.value specified with orient vertical', function() {
```

Comment:
You should test when you map a field to `size` rather than value. 

Also, add another test for horizontal case.